src/App.js

```diff
@@ -22,12 +30,18 @@ export class App extends Component {
                   id="search-query"
                   className="input"
                   placeholder="Type search word"
+                  value={this.state.query}
+                  onChange={(event) => {
+                    this.setState({
+                      query: event.target.value.toLowerCase(),
```

Comment:
bad idea, do not limit the functionality available to the user if the user wants to write using upper case he should be able  to do this

Reply:
use toLowerCase behind the scene, so the user doesn't even notice those actions